nix-installer.nix

```diff
@@ -0,0 +1,172 @@
+{ installationSlug ? "nix-install", installedPackages, postInstall ? null }:
+let
+  pkgs = import (import ./fetchNixpkgs.nix (builtins.fromJSON (builtins.readFile ./nixpkgs-src.json))) { config = {}; overlays = []; };
+  nix-bundle = import (pkgs.fetchFromGitHub {
+    owner = "input-output-hk";
+    repo = "nix-bundle";
+    rev = "630e89d1d16083";
+    sha256 = "1s9vzlsfxd2ym8jzv2p64j6jlwr9cmir45mb12yzzjr4dc91xk8x";
+  }) { nixpkgs = pkgs; };
+  installerBundle = nix-bundle.nix-bootstrap {
+    target = "${installer}";
+    run = "/bin/installer";
+    nixUserChrootFlags = "-c -m /home:/home -p HOME";
+  };
+  nixSrc = pkgs.fetchFromGitHub {
```

Comment:
This can be removed after nixpkgs bump to include Nix 2.0?

Reply:
this is to handle a bug that is in the released version of 2.0, so we have to wait for that fix to hit the nixpkgs daedalus is pinned to